### PR TITLE
docs: mention missing optimized by default packages in `optimizePackageImports`

### PR DESCRIPTION
**Pull request by AhmedBaset**
Source:
https://github.com/vercel/next.js/blob/9986e822637ca70efd4de8e0cb923798b684f4b9/packages/next/src/server/config.ts#L827-L886

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 3b419e09c96d6b4c2703269399e5b9a8538bcc00

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/02-api-reference/05-next-config-js/optimizePackageImports.mdx [hunk 1]
```diff
@@ -17,4 +17,29 @@ module.exports = {
 }
 ```
 
-Libraries like `@mui/icons-material`, `@mui/material`, `date-fns`, `lodash`, `lodash-es`, `react-bootstrap`, `@headlessui/react`, `@heroicons/react`, and `lucide-react` are already optimized by default.
+The following libraries are already optimized by default with zero-config:
```
Comment (leerob) [inline docs/02-app/02-api-reference/05-next-config-js/optimizePackageImports.mdx:20]:
```suggestion
The following libraries are optimized by default:

```

File: docs/02-app/02-api-reference/05-next-config-js/optimizePackageImports.mdx [hunk 2]
```diff
@@ -17,4 +17,29 @@ module.exports = {
...
+- `rxjs`
+- `@mui/material`
+- `@mui/icons-material`
+- `recharts`
+- `react-use`
+- `@material-ui/core`
+- `@material-ui/icons`
+- `@tabler/icons-react`
+- `mui-core`
+- `react-icons/*`
```
Comment (leerob) [inline docs/02-app/02-api-reference/05-next-config-js/optimizePackageImports.mdx:45]:
```suggestion
- `react-icons/*`

```

**Review by leerob — APPROVED** [review]
I can see this going either way – there are advantages for linking back to the source code directly. Regardless, given there's some list here now, it makes sense to keep the list as close as possible.

**Comment by A7med3bdulBaset** [comment]
> I can see this going either way – there are advantages for linking back to the source code directly. Regardless, given there's some list here now, it makes sense to keep the list as close as possible.

I thought about linking the code but ran into a problem. If we link to the current code, it will be outdated after a while because it won't be updated, and if we link to the Canary branch, the line numbers will change after editing the file in later commits.

**Review by leerob — APPROVED** [review]

File: docs/02-app/02-api-reference/05-next-config-js/optimizePackageImports.mdx [hunk 3]
```diff
@@ -17,4 +17,31 @@ module.exports = {
...
+
```
Comment (leerob) [inline docs/02-app/02-api-reference/05-next-config-js/optimizePackageImports.mdx:47]:
```suggestion
```

**Pull request merged**